tests/cases/compiler/useUnknownInCatchVariables01.ts

```diff
@@ -0,0 +1,35 @@
+// @useUnknownInCatchVariables: true

```

Comment:
except for this, I'm not convinced we even need a new test, since we know that `unknown` narrows, and from some other tests, that strict changes the type of the catch variable to unknown.

I guess there's nothing to change here, except possibly dropping the narrowing tests from this test.

Reply:
I added that style of code in because most of the purpose of this feature is to ensure that catch variables are narrowed, and that we error on usages if a user doesn't do so.